### PR TITLE
refactor: use the activeDelegates number provided by the network

### LINKED ISSUE #335: The delegates list does not show for blockchains under 51 delegates.
**Describe the bug**
Currently when asking for the delegates list on a network with under 51 delegates results in an error while visiting the delegate page. This is because the `api/delegates` endpoint errors out when a limit is imposed for more than which is available.

As an example, if I'm using the ARK Deployer to launch a network with 13 delegates, this means that `/api/delegates?limit=51&offset=0` will error me out.

### PR DESCRIPTION
**Pull request by vulet**
## Proposed changes

We refactor the static 51 limit to an activeDelegates limit, which resolves #335.

### CODE REVIEW

File: src/providers/ark-api/ark-api.ts [hunk 1]
```diff
@@ -126,10 +127,13 @@ export class ArkApiProvider {
 
     // Fallback if the fetchEpoch fail
     this._network.epoch = arktsConfig.blockchain.date;
+    // Fallback if the fetchActiveDelegates fail
+    this._network.activeDelegates = 51;
     this.userDataProvider.onUpdateNetwork$.next(this._network);
 
     this.fetchFees().subscribe();
     this.fetchEpoch().subscribe();
+    this.fetchActiveDelegates().subscribe();
```
Comment (luciorubeens) [inline src/providers/ark-api/ark-api.ts]: Don't need to implement a new method. The node configuration has already been fetched to set the vendorField length in the `updateNetwork` method:

https://github.com/ArkEcosystem/mobile-wallet/blob/develop/src/providers/ark-api/ark-api.ts#L471-L476

**Comment by vulet** [comment]
Thanks @luciorubeens. Anything else?

File: src/providers/ark-api/ark-api.ts [hunk 2]
```diff
@@ -126,10 +127,13 @@ export class ArkApiProvider {
 
     // Fallback if the fetchEpoch fail
     this._network.epoch = arktsConfig.blockchain.date;
+    // Fallback if the fetchActiveDelegates fail
+    this._network.activeDelegates = 51;
```
[the remark this answers is not in the document]
Comment (vulet) [reply src/providers/ark-api/ark-api.ts]: Hey, I think the new commits should resolve this. Not sure how to link the commit here.

File: src/providers/ark-api/ark-api.ts (hunk 1 above)
Comment (vulet) [reply src/providers/ark-api/ark-api.ts]: Hey, I think the new commit should resolve this. Not sure how to link the commit here.

**Pull request merged**